src/bit_set.rs

```diff
@@ -15,26 +15,21 @@ use std::slice;
 #[derive(Default,Debug)]
 pub struct Bitset {
     head: usize,
-    tail: Option<Box<Vec<usize>>>,
+    tail: Option<Vec<usize>>,
 }
 
 fn bits_per_word() -> usize {
     usize::max_value().count_ones() as usize
 }
 
-#[inline(never)]
-fn clone_slow(arg: &Box<Vec<usize>>) -> Box<Vec<usize>> {
-    arg.clone()
-}
-
 impl Clone for Bitset {
     #[inline]
     fn clone(&self) -> Bitset {
```

Comment:
Not necessarily, but it does make the binaries larger / make the compile time longer.  Probably much less important for `Bitset` than it is for core types like `Vec`; I don't feel strongly about this.